### PR TITLE
Lies! Nothing but Lies!!!!1!!

### PR DESCRIPTION
**Pull request by stkem**
fake twitter waitlist length and position counts
@andrewconner @ahsu1230

### CODE REVIEW

**Comment by yingjieMiao** [comment]
i hope you guys did some simulations

**Comment by eishay** [comment]
just statistically inaccurate

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/TwitterWaitlistCommander.scala [hunk 1]
```diff
@@ -18,6 +20,9 @@ class TwitterWaitlistCommanderImpl @Inject() (
     twitterWaitlistRepo: TwitterWaitlistRepo,
     clock: Clock) extends TwitterWaitlistCommander with Logging {
 
+  private val WAITLIST_LENGTH_SHIFT = 1152
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/TwitterWaitlistCommander.scala:23]: Looks random to me.

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/TwitterWaitlistCommander.scala [hunk 2]
```diff
@@ -39,5 +44,19 @@ class TwitterWaitlistCommanderImpl @Inject() (
...
+  def getFakeWaitlistPosition(userId: Id[User], handle: String): Option[Long] = {
+    db.readOnlyMaster { implicit session =>
+      twitterWaitlistRepo.getByUserAndHandle(userId, handle).map { waitlistEntry =>
+        twitterWaitlistRepo.countActiveEntriesBeforeDateTime(waitlistEntry.createdAt)
+      }
+    }.map(_ * WAITLIST_MULTIPLIER + WAITLIST_LENGTH_SHIFT)
+  }
+
+  def getFakeWaitlistLength(): Long = {
+    db.readOnlyMaster { implicit session =>
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/TwitterWaitlistCommander.scala]: Replica, perhaps?

Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/TwitterWaitlistCommander.scala]: yeah. changing.

**Pull request merged**